client/__fixtures/query-dead.json

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
Where do we use fixtures?

Reply:
This is temporary, it was necessary for the development of the frontend. I will remove it now